### ISSUE TITLE
Error when trying to install package

### ISSUE THREAD

**Issue by timgivois** [vesln/package#10]
When I try to install package via `npm` or `yarn` I'm getting the following error:
```
verbose stack package: No valid versions available for package
```

node version: v10.1.0
npm version: v6.4.1
os: macOS Mojave 10.14.2

**Comment by pelletiermaxime** [comment]
Yes this package is used by 'temporary' which is used by 130 other packages, so deleting it from npm broke npm install for a lot of users.

**Comment by pelletiermaxime** [comment]
Looks like this happened once before https://status.npmjs.org/incidents/36j9bnllqtnj.

**Comment by s0** [comment]
This seems to be when I changed my username on NPM from `samlanning` to `s0`, Unfortunately it looks like NPM doesn't have good redirect functionality, any package references need to be updated.
